gensim/models/doc2vec.py

```diff
@@ -239,9 +239,9 @@ class TaggedDocument(namedtuple('TaggedDocument', 'words tags')):
     and `tags` (a list of tokens). Tags may be one or more unicode string
     tokens, but typical practice (which will also be most memory-efficient) is
     for the tags list to include a unique integer id as the only tag.
-
+    
```

Comment:
Unrelated change (leading space in empty line), please revert all changes for `doc2vec.py` and Travis will pass successfully